chaos_kitten/brain/attack_planner.py

```
@@ -5,6 +5,15 @@
 from dataclasses import dataclass, field
 from typing import Any, Dict, List, Optional
 
+from typing import Any
```

Ruff (0.15.0)

[error] Redefinition of unused `Any` from line 6: `Any` redefined here

Remove definition: `Any`

(F811)